qt5ct_color_scheme.yapsy-plugin

```diff
@@ -0,0 +1,9 @@
+[Core]
+Name = Qt5ct Color Scheme
+Module = qt5ct_color_scheme
+   
+[Documentation]
+Description = Creates a qt5ct color scheme file in the appropriate folder with the same name as the preset
+Author = Gradience Team
+Version = 0.1.0
+Website = https://gradienceteam.github.io/plugins/firefox-gnome-theme/
```

Comment:
This should be changed to https://github.com/GradienceTeam/Plugins/qt5ct (or similar, because you didn't put this plugin in a folder, as I suggested in chat)